scripts/build/termux_step_get_dependencies.sh

```diff
@@ -106,7 +153,8 @@ termux_run_build-package() {
 		$([[ "${TERMUX_FORCE_BUILD}" == "true" && "${TERMUX_FORCE_BUILD_DEPENDENCIES}" == "true" ]] && echo "-F") \
 		$([[ "${TERMUX_PKGS__BUILD__RM_ALL_PKG_BUILD_DEPENDENT_DIRS}" == "true" ]] && echo "-r") \
 		$([[ "${TERMUX_WITHOUT_DEPVERSION_BINDING}" = "true" ]] && echo "-w") \
-			--format $TERMUX_PACKAGE_FORMAT --library $set_library "${PKG_DIR}"
+		$([[ "${TERMUX_DEPENDENCY_DOWNLOAD_PARALLELIZING-true}" = "false" ]] && echo "--disable-dependency-download-parallelizing") \
```

Comment:
```suggestion
		$([[ "${TERMUX_DEPENDENCY_DOWNLOAD_PARALLELIZING-true}" == "false" ]] && echo "--no-parallel") \
```

Reply:
Same as above, this option name does not specify which parallelizing is to be disabled.